mappings/net/minecraft/world/gen/chunk/ChunkDecorator.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_5444 net/minecraft/world/gen/chunk/ChunkDecorator
```

Comment:
Not a fan of the term Decorator here since it's not really a decorator. Seems more like a facade for accessing surface properties to me, but that can be renamed later.